supervision/metrics/mean_average_precision.py

```diff
@@ -626,3 +229,1253 @@ def plot(self):
 
         plt.tight_layout()
         plt.show()
+
+    def to_pycocotools_output(self) -> str:
```

Comment:
`MeanAveragePrecisionResult` currently have `__str__`. I think we can replace `__str__` implementation with this.

Reply:
Done!